mittab/libs/data_import/import_rooms.py

```diff
@@ -48,37 +49,49 @@ def import_rooms(fileToImport):
             return room_errors
 
     for i in range(1, num_rooms):
+
+        # headers are
+        # name, room rank
+
         room_name = sh.cell(i, 0).value
         if room_name == '':
-            room_errors.append("Row " + str(i) + ": Empty Room Name")
-            continue
-        try:
-            Room.objects.get(name=room_name)
-            room_errors.append(room_name + ': Duplicate Room Name')
+            room_errors.append("Row " + str(i) + ": Empty room name")
             continue
-        except:
-            pass
 
-        #Load and validate room_rank
+        duplicate = False
+        if Room.objects.filter(name=room_name).first() is not None:  # check for duplicates
```

Comment:
No need to do this, actually. The model already validates this: https://github.com/jolynch/mit-tab/blob/master/mittab/apps/tab/models.py#L133